### ISSUE TITLE
Create a mechanism to query supdoc and supdocfolder

### ISSUE THREAD

**Issue by craibuc** [Intacct/intacct-sdk-net#146]
There does not appear to be a way to send a `get_list` action via the .NET SDK:

```
<get_list object="supdocfolder"></get_list>
```
or
```
<get_list object="supdoc" maxitems="20">
    <filter>
        <expression>
            <field>recordno</field><operator>=</operator><value>38</value>
        </expression>
    </filter>
    <fields>
        <field>supdocid</field>
        <field>supdocname</field>
    </fields>
    <sorts><sortfield order='asc'>supdocname</sortfield></sorts>
</get_list>
```

**Comment by craibuc** [comment]
Created pull request #152

**Comment by karstenintacct** [comment]
Hi Craig,
Currently, this is not yet available through the SDK. I am wondering if you have already implemented this change and would want to create a pull request for it. I am working in the meantime with an Engineer to look into it.

thank you 
Karsten

**Comment by karstenintacct** [comment]
Hi Craig,
We are taking a look at your pull request and try to merge this in.

Thank you
Karsten

**Comment by craibuc** [comment]
Thank you.

**Comment by dylan-holt-sage** [comment]
Merged code as part of PR #175 , added support for logical filtering on top of changes in PR #152. Thanks for your contributions Craig! Closing this issue.
